synapse/lib/node.py

```diff
@@ -207,8 +207,7 @@ async def walk(n, p):
 
             embdnode = retn.get(nodepath)
             if embdnode is None:
-                embdnode = retn[nodepath] = {}
-                embdnode['*'] = s_common.ehex(node.buid)
+                embdnode = retn[nodepath] = {'*': s_common.ehex(node.buid), '*form': node.form.name}
```

Comment:
Would it make sense to make these `$iden` and `$form` to align a bit with the dict ctor "out of band" data syntax we're using?

Reply:
Yeah that would be a good idea, I think we still need to keep the `*` key with iden for it to be backwards compatible in 2x though.

Reply:
:+1: Lets go with "both" for now and carve out the `*` in 3.x